src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -81,6 +86,11 @@
     <ProjectReference Include="..\..\..\Workspaces\VisualBasic\Portable\Microsoft.CodeAnalysis.VisualBasic.Workspaces.vbproj" />
     <ProjectReference Include="..\..\Core\Portable\Microsoft.CodeAnalysis.Features.csproj" />
 
+    <!-- Make sure we include the RazorCompiler ExternalAccess since this needs to be deployed with the compiler for things that might
+         run the Razor generator. We bundle this in the MWorkspaces.MSBuild NuGet package directly, but since we're consuming a project reference
+         rather than a package reference we'll have to include this here. -->
+    <ProjectReference Include="..\..\..\Tools\ExternalAccess\RazorCompiler\Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler.csproj" />
```

Comment:
💡 If this project produces a nupkg, consider adding `PrivateAssets="all"` to any `<PackageReference>` items that are embedded as binaries inside the nupkg.

Reply:
We'd want these references to also flow to the unit test project so that'd just make that harder.